### ISSUE TITLE
Is there a collection or library of commonly used loss functions?

### ISSUE THREAD

**Issue by asmith26** [google/jax#4215]
Hi there,

Are there any existing libraries for importing commonly used loss functions? This could be similar to e.g.  https://scikit-learn.org/stable/modules/classes.html#sklearn-metrics-metrics

I see there are some e.g. optimizers included with jax: https://jax.readthedocs.io/en/latest/jax.experimental.optix.html

I have been thinking about creating a library for commonly used loss functions (probably based heavily on the scikit-learn ones), but just thought I'd ask in case I'll be re-implementing something that already exists.

Thanks for this awesome lib! :)

**Comment by asmith26** [comment]
As a bit of a learning excercise (for jax and loss functions), I've ended up creating https://github.com/asmith26/jax_toolkit

Any suggestions/feedback (e.g. mistakes, performance improvements, contributions,...) very welcome :)
